impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -134,7 +140,7 @@ private static void prepareLogDirsForKraft(String clusterId, KafkaConfig config,
                     boolean.class);
             // note ignoreFormatter=true so tolerate a log directory which is already formatted. this is
             // required to support start/stop.
-            formatCommand.invoke(null, LOGGING_PRINT_STREAM, directories, metaProperties, MINIMUM_BOOTSTRAP_VERSION, true);
+            formatCommand.invoke(null, LOGGING_PRINT_STREAM, directories, metaProperties, metadataVersion, true);
```

Comment:
It would be possible to inject the SCRAM users in at storage formatting.  This is the approach that the [kafka-native](https://github.com/ozangunalp/kafka-native/blob/fc0d173d95d5edc61705307aec6f7e83769c7d17/kafka-server/src/main/java/com/ozangunalp/kafka/server/EmbeddedKafkaBroker.java#L224) now takes.  I decided against this for the InVM path.  My motivation - it would have bifurcated with different code for the Zookeeper path and KRaft paths.